configuration/ibm/50001001.json

```diff
@@ -1550,21 +1555,28 @@
         "/sys/bus/i2c/drivers/at24/4-0050/eeprom": [
             {
                 "inventoryPath": "/system/chassis/motherboard/pcieslot0/pcie_card0",
-                "devAddress": "4-0050",
                 "pcaChipAddress": "4-0060",
```

Comment:
why is not under a tag ?

Reply:
This is not related to processing of pre/post so did not change that here. As and when we handle these information, will see if they can be grouped under any tag.